scripts/e2e/config/minio/values.yaml

```diff
@@ -0,0 +1,30 @@
+replicas: 4
```

Comment:
Why 4 replicas? Previously, we only use 1 replica for minio.

Reply:
I don't really know the details with new minio helm chart, but it now needs at least 4 replicas to work.